.kitchen.yml

```diff
@@ -6,7 +6,7 @@ driver:
 
 provisioner:
   name: chef_zero
-  require_chef_omnibus: 11.16.4
+  require_chef_omnibus: 12.1.0
```

Comment:
TBH we should just lock to the value we are currently using in our CI infrastructure which is `12.0.3`.